lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -499,9 +499,12 @@ export class UserAgentApplication {
      *
      */
     private async acquireTokenHelper(account: Account, interactionType: InteractionType, isLoginCall: boolean, request: AuthenticationParameters, resolve?: any, reject?: any): Promise<void> {
+        this.logger.verbose("AcquireTokenHelper has been called");
+
```

Comment:
I think a log message indicating the value of `isLoginCall` (which is not PII) would be helpful.

Reply:
Along with `interactionType`

Reply:
+1

Reply:
Done